lab-justin/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "lab-justin",
+  "version": "1.0.0",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "author": "justin",
+  "license": "ISC",
+  "dependencies": {
+    "cowsay": "^1.1.6"
+  },
+  "devDependencies": {
+    "gulp": "^3.9.1",
+    "gulp-eslint": "^2.0.0",
+    "gulp-watch": "^4.3.5"
```

Comment:
This is something else--gulp already has a built in watch task, so you don't need to install this too.